duckduckgo/iOS
# remove main view controller storyboard

Task/Issue URL: https://app.asana.com/0/0/1205584556126567/f
Tech Design URL:
CC:

**Description**:

Note: This PR is based on #2045.

**Steps to test this PR**:

1. Clean install of the app, get through onboarding and check all buttons on the main screen work as expected
4. Browse to a site.  Check all buttons work as expected
5. Clear tabs and data. Check UI looks right and buttons work as expected
7. On iPad specifically in full screen and split screen configurations check UI looks and behaves as expected
8. Check launching app with home screen showing
9. Check launching app with tab open
10. Edit HomeRowReminder.swift and make 'showNow()' return true.  Open a new tab and perform a search.  Ensure home row banner is shown.

**Orientation Testing**:


**Device Testing**:


**OS Testing**:


**Theme Testing**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
